src/app/core/site-navbar/_shared.scss

```diff
@@ -17,7 +17,7 @@ $nav-element-hover-color: $ux-color-white !default;
 $nav-element-hover-background-color: $ux-color-primary-medium !default;
 
 $nav-element-active-background-color: $ux-color-primary-dark !default;
-$nav-element-active-color: $ux-color-highlight !default;
+$nav-element-active-color: $ux-color-brand-background !default;
```

Comment:
I believe this change should be handled as an override in your teams downstream project.
Changing the default will effect all downstream projects.

Reply:
I agree! Fixed.